iOS/DuckDuckGo/DataBrokerProtectionDebugViewController.swift

```diff
@@ -71,39 +90,41 @@ final class DataBrokerProtectionDebugViewController: UITableViewController {
 
     override func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
         switch Sections(rawValue: section) {
-        case .databaseBrowser: return 1
+        case .database: return DatabaseRows.allCases.count
         case .none: return 0
 
         }
     }
 
     override func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
         switch Sections(rawValue: indexPath.section) {
-        case .databaseBrowser:
-            didSelectDatabaseBrowser(at: indexPath)
+        case .database:
+            didSelectDatabase(at: indexPath)
         case .none:
             break
         }
 
         tableView.deselectRow(at: indexPath, animated: true)
     }
 
-    private func configure(_ cell: UITableViewCell, forDebugFeatureAtRow row: Int) {
-        switch row {
-        case 0:
-            cell.textLabel?.text = "Database browser"
-        default:
-            break
-        }
-    }
-
-    private func didSelectDatabaseBrowser(at indexPath: IndexPath) {
+    private func didSelectDatabase(at indexPath: IndexPath) {
         guard let dbpManager = DataBrokerProtectionIOSManager.shared else {
```

Comment:
This is something to undo, but I still haven't decided on the best way to do it. I can think of two options:
- Pass it in from the service, to the MainCoordiantor, MainViewController, Settings, etc
- Put it on AppDependencyProvider but init it with the rest of the AppConfiguration things

I don't like either tbh. The constructor for MainViewController is already absurd, and it has no business with this class anyway. The latter I think is acceptable since I think dbpManager should be optional anyway, but it feels a bit like the pixelkit thing where it's reliant on setup happening later, which is a bit of a foot gun